.github/workflows/lambda-functions.yml

```diff
@@ -91,6 +52,22 @@ jobs:
         with:
           python-version: "3.8"
 
+      - name: Package and deploy user-flows lambda function
+        if: ${{ (inputs.lambdaName == 'UserFlows') || (inputs.lambdaName == 'All') }}
+        run: |
+          cd user_flows/
+          python3 -m venv venv_user_flows
+          source venv_user_flows/bin/activate
+          pip install -r requirements-lambda.txt
+          deactivate
+          cat ../../certs/VA-Internal-S2-ICA11.cer >> ./venv_user_flows/lib/python3.8/site-packages/certifi/cacert.pem
+          cat ../../certs/VA-Internal-S2-RCA2.cer >> ./venv_user_flows/lib/python3.8/site-packages/certifi/cacert.pem
```

Comment:
These lines are the operationally significant change.  The rest is tech debt elimination.